lib/helpers.bash

```diff
@@ -951,6 +951,24 @@ function pathmunge() {
 	fi
 }
 
+function _remote_name() {
```

Comment:
I'm not sure how much the contribution guideline is strictly applied to the project, but [Coding Style - `docs/contributing.rst`](https://github.com/Bash-it/bash-it/blob/4c5ac697f593169ab09a63e0f78f85a20d01c47a/docs/contributing.rst#code-style) says

> - When creating new functions, please use a dash ("-") to separate the words of the function's name, e.g. `my-new-function`. Don't use underscores, e.g. `my_new_function`.
> - Internal functions that aren't to be used by the end user should start with an underscore, e.g. `_my-new-internal-function`.

Reply:
yup, also maybe call it _get-git-default-remote-name is a bit more descriptive, and add a remark to doc the function.